### ISSUE TITLE
tpm2_checkquote manpage out of date

### ISSUE THREAD

**Issue by dnoliver** [tpm2-software/tpm2-tools#1488]
tpm2_checkquote manpage do not match the options in the code 

Code:
```
    const struct option topts[] = {
            { "halg",           required_argument, NULL, 'g' },
            { "message",        required_argument, NULL, 'm' },
            { "format",         required_argument, NULL, 'f' },
            { "sig",            required_argument, NULL, 's' },
            { "pcr-input-file", required_argument, NULL, 'F' },
            { "pubfile",        required_argument, NULL, 'u' },
            { "qualify-data",   required_argument, NULL, 'q' },
    };
```

Man:
```
[intel@localhost tpm2-tools]$ cat ./man/tpm2_checkquote.1.md | grep -e "--"
  * **-c**, **\--key-context**=_KEY\_CONTEXT\_OBJECT_:
  * **-G**, **\--halg**=_HASH\_ALGORITHM_:
  * **-m**, **\--message**=_MSG\_FILE_:
  * **-s**, **\--sig**=_SIG\_FILE_:
  * **-f**, **\--format**:
  * **-p**, **\--pcrs**:
  * **-q**, **\--qualify-data**:
```

1. `-c, --key-context` was removed
2. `-G, --halg` is `-g` in the code. Should this be `-G` for consistency with `tpm2_quote`?
3. `-s, --sig` works, but it is `--signature` in the `tpm2_quote` command (inconsistent)
4. `-p, --pcrs` is `-F, --pcr-input-file` in the code, should be `-p, --pcrs` for consistency with `tpm2_quote`
5. `-u, --pubfile` is not documented in the man

**Comment by idesai** [comment]
There is an existing issue #1042  for auditing all the man pages

**Comment by joholl** [comment]
With 31bcf88, the options of `tpm2_checkquote.c` were refactored:
 * `-g`/`--halg` became `-G`/`--halg`
 * `-c`/`--pubkey` became `-u`/`--pubfile`
 * `-p`/`--pcrs` became `-F`/`--pcr-input-file`

However, the corresponding man page was not updated which lead to inconsistent documentation. This was fixed in 30f62700.

For the remaining issue: I agree that `--sig` (`tpm2_checkquote.c`) and `--signature` (`tpm2_quote.c`) should be aligned.
